tests/Sdk/Acceptance/Presentation/RestApi/Controller/v1/SdkInitProjectControllerCest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Acceptance\Presentation\RestApi\Controller\v1;
+
+use SprykerSdk\Sdk\Presentation\RestApi\Enum\OpenApiField;
+use SprykerSdk\Sdk\Tests\AcceptanceTester;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group Sdk
+ * @group Acceptance
+ * @group Presentation
+ * @group RestApi
+ * @group Controller
+ * @group v1
+ * @group SdkInitProjectControllerCest
+ * Add your own group annotations below this line
+ */
+class SdkInitProjectControllerCest
+{
+    /**
+     * @var string
+     */
+    protected const ENDPOINT = '/sdk-init-project';
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function iSeeJsonResponseAfterCallSdkInitProjectEndpoint(AcceptanceTester $I): void
```

Comment:
can you add please one more case for project init fail? For example, if the given path is not accessible for some reason.

Reply:
we don't have any path here. will add for wrong body structure